vcpkg/get_started/get-started-linux.md

```diff
@@ -0,0 +1,157 @@
+---
+title: Install and use packages on Linux
+description: Tutorial guides the user through the process of installing and using packages on Linux with vcpkg.
+author: JavierMatosD
+ms.date: 08/23/2023
+ms.author: javiermat
+ms.prod: vcpkg
+---
+# Install and use packages on Linux
+
+## Description
+
+This tutorial guides you through setting up a new C++ "Hello World" project using the `fmt` library. You'll manage dependencies, build the application, and eventually run your project, all using vcpkg. This tutorial assumes you have vcpkg setup and ready to use. If not, check out our initial setup [tutorial](setup-vcpkg.md).
+
+### Prerequisites
+
+For this tutorial, you'll need to have `git`, `bash`, `CMake`, and a C++ compiler installed.
+
+## Setup the project
+
+Create a directory for your new project next to your vcpkg clone and navigate into it:
+
+```bash
+mkdir helloworld && cd helloworld
+```
+
+Initiate a new project within this directory, creating the manifest (`vcpkg.json`) and default configuration files (`vcpkg-configuration.json`). From within the `helloworld` directory, run:
+
+```bash
+..\vcpkg\vcpkg new --name helloworld --version 1.0
+```
+
+>[!NOTE]
+>Ensure the project directory name matches exactly with the name you give to the `vcpkg new` command. This ensures the manifest files land in the correct directory.
+
+As mentioned above, the `vcpkg new` command will add a `vcpkg.json` file and a `vcpkg-configuration.json` file to your projects directory. The `vcpkg.json` file should look like this:
+
+```json
+{
+    "name": "helloworld",
+    "version": "1.0"
+}
+```
+
+This `vcpkg.json` file serves as the foundation for your vcpkg-based C++ project. It identifies your project by name and version, setting the stage for additional attributes and dependencies that might be added as your project grows.
+
+To learn more about `vcpkg.json`, check out our reference [documentation](..\reference\vcpkg-json.md).
+
+The `vcpkg-configuration.json` file serves a different purpose. This file essentially allows you to specify which registry vcpkg should use to look for packages (both by default and additionally). It should look like this:
```

Comment:
```suggestion
The `vcpkg-configuration.json` file serves a different purpose. This file allows full control over your dependencies' sources and it's out of scope for this tutorial. 

For this tutorial, our dependencies all come from the vcpkg registry at <https://github.com/Microsoft/vcpkg>. When there's no explicit configuration provided, vcpkg uses this registry as default. So the next step is to delete this file.

`rm vcpkg-configuration.json`
```

For this tutorial we should probably avoid overwhelming users with this information, let's just give a brief explanation and have them delete the `vcpkg-configuration.json` file.